doc/task_breakdown.md

```diff
@@ -0,0 +1,61 @@
+# SQL Query Builder Task Breakdown
+
+## Task 1: Basic structure and first working query (2 days)
+- Implement a basic UI layout with components: Tables, Columns, Filters, Sorting, Joins
+- Implement functionality to display a list of available database tables
+- Implement selection of a single table for query execution
+- Implement execution of SELECT * FROM table for the selected table
+- Display the query result in a table format
+- Implement basic error handling for missing table selection
+
+Expected result:
+- The user sees the UI, can select a table, and retrieve all its data using SELECT * FROM table
+- If no table is selected, an error message is shown
+
+---
+
+## Task 2: Selecting columns (1 day)
+- Implement display of available columns for selected tables
+- Add functionality to select specific columns for query execution
+- Implement execution of queries with selected columns (SELECT column1, column2 FROM table)
+- Implement error handling for missing column selection
```

Comment:
how it can be missed? you just select it from the table...
if you want to handle situation when no column selected, you can just disable output before selecting column, but this is not an ERROR